src/markdown/tutorial/part-1/05-more-about-components.md

```diff
@@ -23,63 +23,64 @@ Let's start by creating the `<Rental>` component. This time, we will use the com
 
 ```run:command hidden=true cwd=super-rentals
 ember test --path dist
-git add app/components/rental.hbs
-git add tests/integration/components/rental-test.js
+git add app/components/rental.gjs
+git add tests/integration/components/rental-test.gjs
 ```
 
-We will start by editing the template. Let's *[hard-code](https://en.wikipedia.org/wiki/Hard_coding)* the details for one rental property for now, and replace it with the real data from the server later on.
-
-```run:file:patch lang=handlebars cwd=super-rentals filename=app/components/rental.hbs
-@@ -1,1 +1,17 @@
--{{yield}}
-\ No newline at end of file
-+<article class="rental">
-+  <div class="details">
-+    <h3>Grand Old Mansion</h3>
-+    <div class="detail owner">
-+      <span>Owner:</span> Veruca Salt
+We will start by editing the component template. Let's *[hard-code](https://en.wikipedia.org/wiki/Hard_coding)* the details for one rental property for now, and replace it with the real data from the server later on.
+
+```run:file:patch lang=gjs cwd=super-rentals filename=app/components/rental.gjs
+@@ -1,3 +1,19 @@
+ <template>
+-  {{yield}}
++  <article class="rental">
++    <div class="details">
++      <h3>Grand Old Mansion</h3>
++      <div class="detail owner">
++        <span>Owner:</span> Veruca Salt
++      </div>
++      <div class="detail type">
++        <span>Type:</span> Standalone
++      </div>
++      <div class="detail location">
++        <span>Location:</span> San Francisco
++      </div>
++      <div class="detail bedrooms">
++        <span>Number of bedrooms:</span> 15
++      </div>
 +    </div>
-+    <div class="detail type">
-+      <span>Type:</span> Standalone
-+    </div>
-+    <div class="detail location">
-+      <span>Location:</span> San Francisco
-+    </div>
-+    <div class="detail bedrooms">
-+      <span>Number of bedrooms:</span> 15
-+    </div>
-+  </div>
-+</article>
++  </article>
+ </template>
 ```
 
 Then, we will write a test to ensure all of the details are present. We will replace the boilerplate test generated for us with our own assertions, just like we did for the `<Jumbo>` component earlier:
 
-```run:file:patch lang=js cwd=super-rentals filename=tests/integration/components/rental-test.js
-@@ -8,18 +8,11 @@
-
+```run:file:patch lang=gjs cwd=super-rentals filename=tests/integration/components/rental-test.gjs
+@@ -8,20 +8,11 @@ module('Integration | Component | rental', function (hooks) {
+ 
 -  test('it renders', async function (assert) {
--    // Set any properties with this.set('myProperty', 'value');
--    // Handle any actions with this.set('myAction', function(val) { ... });
--
--    await render(hbs`<Rental />`);
+-    // Updating values is achieved using autotracking, just like in app code. For example:
+-    // class State { @tracked myProperty = 0; }; const state = new State();
```

Comment:
depending on when this update lands, we can use:
```js
import { trackedObject } from '@ember/reactive/collections';

const state = trackedObject({ myProperty: 0 });
```

Looks like this'll be landing in 6.8